### ISSUE TITLE
[Lab 3] Βήμα 8

### ISSUE THREAD

**Issue by pmisiakos** [slp-ntua/patrec-labs#74]
Καλησπέρα σας, 

Στο ερώτημα 8 ζητάτε να εκπαιδεύσουμε 3 διαφορετικά μοντέλα (ένα για κάθε χαρακτηριστικό valence, energy, danceability)?

Δηλαδή στο βήμα 8e θα πάρουμε την κάθε πρόβλεψη από το εκάστοτε μοντέλο και θα τις συνδυάσουμε για να βρούμε το spearman correlation?

Με εκτίμηση,
Παναγιώτης Μισιακός.

**Comment by filby89** [comment]
Σωστά, θα αξιολογήσετε το κάθε μοντέλο χρησιμοποιώντας τις προβλέψεις του και να βρείτε το spearman correlation με τις ground truth.

**Comment by michaelbzms** [comment]
Για το Early stopping να σταματήσουμε βάσει του loss ή θα είχε περισσότερο νόημα να σταματήσουμε στο καλύτερο Spearman correlation;

**Comment by georgepar** [comment]
Το τίμιο και σωστό είναι loss.
